.vscode/tasks.json

```diff
@@ -0,0 +1,37 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"label": "build",
+			"command": "make",
+			"args": ["all", "CONFIGURATION=Debug"],
+			"windows": {
+				"command": "make.cmd"
+			}
+		},
+		{
+			"label": "Run Utility",
+			"command": "dotnet ${workspaceRoot}/engine/bin/OpenRA.Utility.dll ${input:modId} ${input:command}",
+			"type": "shell",
+			"options": {
+				"env": {
+					"ENGINE_DIR": "${workspaceRoot}/engine",
+					"MOD_SEARCH_PATHS": "${workspaceRoot}/mods,${workspaceRoot}/engine/mods"
+				}
+			}
+		}
+	],
+	"inputs": [
+		{
+			"id": "modId",
+			"description": "ID of the mod to run",
```

Comment:
This repository only has one mod.

Reply:
You can also run commands on the base mods, although I don't know how relevant that would be for this mod in particular.